conf.py

```diff
@@ -47,7 +47,7 @@
 
 # General information about the project.
 project = u'Syncthing'
-copyright = u'2015, The Syncthing Authors'
+copyright = u'2017, The Syncthing Authors'
```

Comment:
Done in https://github.com/syncthing/docs/commit/6b4203296b3b6b62e1bb29ec75ea181d56c7611e, and really sorry for this mess...